tokenizers/src/pre_tokenizers/sequence.rs

```diff
@@ -3,6 +3,7 @@ use crate::tokenizer::{PreTokenizedString, PreTokenizer, Result};
 use serde::{Deserialize, Serialize};
 
 #[derive(Clone, Debug, Serialize, Deserialize)]
+#[cfg_attr(test, derive(PartialEq))]
```

Comment:
Why not add it directly to `derive` in all cases ?

Reply:
Because it wasn't and I imagined it was intended.

Reply:
Nope, I usually added them only when needed, feel free to include it directly!